### ISSUE TITLE
Put item to the front of access time tracking list when getting in LRU

### ISSUE THREAD

**Issue by andyxning** [bluele/gcache#4]
I found that the [`get`](https://github.com/bluele/gcache/blob/master/lru.go#L65-L80) function in `LRU` may be not proper implemented. 

```go
	c.mu.RLock()
	item, ok := c.items[key]
	c.mu.RUnlock()

	if ok {
		it := item.Value.(*lruItem)
		if !it.IsExpired(nil) {
			c.mu.Lock()
			defer c.mu.Unlock()
			c.evictList.MoveToFront(it)
			return it.value, nil
		}
		c.mu.Lock()
		c.removeElement(item)
		c.mu.Unlock()
	}
```

I think `LRU` should move the accessed item when getting to the front of the access tracking list, not just when setting.

**Comment by bluele** [comment]
@andyxning  Thanks! You're right, I'll fix this issue.

**Comment by bluele** [comment]
fix this issue at 310d5ac543a955e07bfbd1de0a5c3a7c6f389d98.
